src/saveload/gamelog_sl.cpp

```diff
@@ -328,6 +328,7 @@ class SlGamelogAction : public DefaultSaveLoadHandler<SlGamelogAction, LoggedAct
 
 		size_t length = SlGetStructListLength(UINT32_MAX);
 		la->change = ReallocT(la->change, length);
+		la->changes = (uint32)length;
```

Comment:
But wow that's a bad variable name, `changes`, especially right next to one named `change`.